README.md

```diff
@@ -25,6 +25,14 @@ in this [blog post](https://blog.nlnetlabs.nl/testing-the-waters-with-krill/).
 
 # Changelog
 
+## 0.12.3 'Sakura'
+
+This release contains some features which were cherry picked from the upcoming
+release (0.13.0) that will enable Publication Server operators to remove
```

Comment:
Do operators care if it is cherry picked or not? Maybe simpler to leave that out?

Reply:
Well otherwise 'Sakura' makes less sense ;) But I updated the text a bit